src/DependencyInjection/SonataUserExtension.php

```diff
@@ -178,7 +178,15 @@ public function configureGoogleAuthenticator($config, ContainerBuilder $containe
         }
 
         $container->setParameter('sonata.user.google.authenticator.forced_for_role', $config['google_authenticator']['forced_for_role']);
-        $container->setParameter('sonata.user.google.authenticator.ip_white_list', $config['google_authenticator']['ip_white_list']);
+
+        if (array_key_exists('ip_white_list', $config['google_authenticator']) && array_key_exists('trusted_ip_list', $config['google_authenticator'])) {
+            throw new \RuntimeException('Please use only ``trusted_ip_list`` parameter. ``ip_white_list`` is deprecated.');
+        }
+        if (array_key_exists('ip_white_list', $config['google_authenticator']) && !array_key_exists('trusted_ip_list', $config['google_authenticator'])) {
+            $container->setParameter('sonata.user.google.authenticator.trusted_ip_list', $config['google_authenticator']['ip_white_list']);
+        } else {
```

Comment:
You should add a comment
`// NEXT_MAJOR: Remove this checks and only set the trusted_ip_list.`

That's how we deal todos.

Reply:
Done